frontend/layouts/static-page/footer/component.tsx

```diff
@@ -216,14 +263,18 @@ export const Footer: React.FC<FooterProps> = ({
                     </Link>
                   </li>
                   <li>
-                    <a className="text-gray-400 disabled:pointer-events-none">
-                      <FormattedMessage defaultMessage="FAQ’s" id="qIvPIE" />
-                    </a>
+                    <Link href={Paths.ComingSoon} passHref>
+                      <a className="text-gray-400 disabled:pointer-events-none">
+                        <FormattedMessage defaultMessage="FAQ’s" id="qIvPIE" />
+                      </a>
+                    </Link>
```

Comment:
With #428 merged, we now have an FAQ page to link to. 
Also, the newly added links' styling doesn't match the other links in the footer (colour, underline when hovered)